### DISCUSSION TITLE
Combining two seawater velocity readers

### DISCUSSION THREAD

**Discussion by eroome** [OpenDrift/opendrift#1299] [General]
Hi,

Is there a way to read two datasets which contain x and y seawater velocity and then use the sum of the two velocities transport the particles? 

cheers,
Ed

**Comment by gauteh** [comment]
Yes, we are in fact experimenting with this at the moment. We have operators on readers, but it is not much tested yet: https://github.com/OpenDrift/opendrift/tree/master/opendrift/readers/operators .
